src/style.css

```
@@ -0,0 +1,94 @@
+:root {
+  --padding-left-big-search: 62px;
+  --padding-left-small-search: 33px;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.form {
+  position: relative;
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.forms-wrapper {
+  margin-top: 20px;
+}
+
+.label-search-icon {
+  position: absolute;
+  top: 46%;
+  left: calc(var(--padding-left-big-search) / 2 + 1px);
+  transform: translateY(-30%) translateX(-30%);
+  cursor: text;
+}
+
+.icon {
+  width: 19px;
+  height: 19px;
+}
+
+.search {
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  height: 100%;
+  border-radius: 4px;
+  padding-left: var(--padding-left-big-search);
+  font-weight: 300;
+  font-size: 16px;
+  color: black;
+  font-family: Avenir, sans-serif;
+  margin-bottom: 20px;
+}
```

Add box-shadow to this element, as it is also set in the designs.
```suggestion
  box-shadow: 0px 1px 8px rgba(61, 78, 97, 0.1)
  }
```

---

I added box-shadow.